### DISCUSSION TITLE
Disabling localization for some routes

### DISCUSSION THREAD

**Discussion by ghost** [sveltekit-i18n/lib#147] [Q&A] [answered]
My configuration is the same as in the  [locale-router](https://github.com/sveltekit-i18n/lib/tree/master/examples/locale-router) example.

I am having problems trying to ignore some routes, no matter what I do the lib always adds the locale to the route and does a redirect.

I was wondering if there is a simple way to prevent a pathname and all the subdirectories from being localized, for example `/blog/*`.

**Comment by jarda-svoboda** [comment]
You could edit your `hooks.server.js` to manage redirects..

**Reply by ghost** [reply, marked as answer]
Yes, I thought there was some function or some other setting, sorry for the dumb question!

I ended up doing something like this:

```javascript
if (pathname.indexOf('blog') > -1) {
  return resolve({ ...event });
}
```

[a comment by an automated account is not shown]
